spark/spark-2.3/src/test/scala/org/apache/spark/sql/hive/XGBoostSuite.scala

```diff
@@ -77,6 +77,7 @@ final class XGBoostSuite extends VectorQueryTest {
         val model = hiveContext.sparkSession.read.format("libxgboost").load(tempDir)
         val predict = model.join(mllibTestDf)
           .xgboost_predict($"rowid", $"features", $"model_id", $"pred_model")
```

Comment:
Let's disable xgboost for spark-2.3.

Reply:
BTW, could you paste Stacktrace of the exception?

Reply:
Since it seems the test fails in JNI, we have no stacktrace;
```
XGBoostSuite:
- resolve libxgboost
- check XGBoost options
AssertError:read can not have position excceed buffer length
...
```